ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlans.j2

```diff
@@ -1,16 +1,12 @@
 {# eos - VLANs #}
-{% if vlans is defined and vlans is not none %}
-{%     for vlan in vlans | arista.avd.natural_sort %}
+{% for vlan in vlans | arista.avd.natural_sort %}
 !
 vlan {{ vlan }}
    name {{ vlans[vlan].name }}
-{%         if vlans[vlan].state is defined and vlans[vlan].state == "suspend" %}
+{%     if vlans[vlan].state is arista.avd.defined %}
    state {{ vlans[vlan].state }}
-{%         endif %}
-{%         if vlans[vlan].trunk_groups is defined %}
-{%             for trunk_group in vlans[vlan].trunk_groups | arista.avd.natural_sort %}
+{%     endif %}
+{%     for trunk_group in vlans[vlan].trunk_groups | arista.avd.natural_sort %}
    trunk group {{ trunk_group }}
-{%             endfor %}
-{%         endif %}
 {%     endfor %}
-{% endif %}
\ No newline at end of file
+{% endfor %}
```

Comment:
missing newline at the end of the file

Reply:
Fixed with commit d42782a527b53acfb04121f714f34954f98cf806